source/features/filter-comments-by-you.tsx

```diff
@@ -1,21 +1,38 @@
 import React from 'dom-chef';
 import select from 'select-dom';
 import features from '../libs/features';
-import {getRepoURL} from '../libs/utils';
+import {getUsername, getRepoURL} from '../libs/utils';
+import {isEnterprise} from '../libs/page-detect';
 
 const repoUrl = getRepoURL();
 
 function init(): void {
-	select('.subnav-search-context .SelectMenu-list a:last-child')!
```

Comment:
```js
select([
	'.subnav-search-context .SelectMenu-list a:last-child',
	'.subnav-search-context details-menu li:last-child' // Old selector, for GHE
])! 
```

Reply:
not quite, as we would miss the `<li>` in case of the old version, i will provide a solution, that works for both, and we can continue the discussion on the new solution :)